docs/book/src/topics/aad-integration.md

```diff
@@ -0,0 +1,143 @@
+# AAD Integration
+
+CAPZ can be configured to use Azure Active Directory (AD) for user authentication. In this configuration, you can log into a CAPZ cluster using an Azure AD token. Cluster operators can also configure Kubernetes role-based access control (Kubernetes RBAC) based on a user's identity or directory group membership.
+
+<aside class="note warning">
+
+<h1> Warning </h1>
+
+The following operations to configure Azure AD applications must be completed by an Azure tenant administrator.
+
+</aside>
+
+## Create Azure AD server component
+
+### Create the Azure AD application
+
+```bash
+export CLUSTER_NAME=my-aad-cluster
+```
+
+```bash
+export AZURE_SERVER_APP_ID=$(az ad app create \
+    --display-name "${CLUSTER_NAME}Server" \
+    --identifier-uris "https://${CLUSTER_NAME}Server" \
+    --query appId -o tsv)
+```
+
+### Update the application group membership claims
+```bash
+az ad app update --id ${AZURE_SERVER_APP_ID} --set groupMembershipClaims=All
+```
+
+## Create Azure AD client component
+```bash
+AZURE_CLIENT_APP_ID=$(az ad app create \
+    --display-name "${CLUSTER_NAME}Client" \
+    --native-app \
+    --reply-urls "https://${CLUSTER_NAME}Client" \
+    --query appId -o tsv)
+```
+
+### Grant the application API permissions
+```bash
+oAuthPermissionId=$(az ad app show --id ${AZURE_SERVER_APP_ID} --query "oauth2Permissions[0].id" -o tsv)
+az ad app permission add --id ${AZURE_CLIENT_APP_ID} --api ${AZURE_SERVER_APP_ID} --api-permissions ${oAuthPermissionId}=Scope
+az ad app permission grant --id ${AZURE_CLIENT_APP_ID} --api ${AZURE_SERVER_APP_ID}
```

Comment:
opened https://github.com/Azure/azure-cli/issues/16862 because I've been getting errors running this command